### PR TITLE
Snake Game is Added

### PR DESCRIPTION
**Pull request by Subhradeep10**
I have added a New snake game in your repo
image

Here's the game
image

### CODE REVIEW

File: Snakegame/snake.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+// Game Constants & Variables
+let inputDir = {x: 0, y: 0}; 
+const foodSound = new Audio('/assets/food.mp3');
+const gameOverSound = new Audio('/assets/gameover.mp3');
+const moveSound = new Audio('/assets/move/mp3');
```
Comment (geekymeeky) [inline Snakegame/snake.js:5]: The path should be `/assets/move.mp3`.

File: Snakegame/snake.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+function isCollide(snake) {
+    // If you bump into yourself 
+    for (let i = 1; i < snakeArr.length; i++) {
+        if(snake[i].x === snake[0].x && snake[i].y === snake[0].y){
+            return true;
+        }
+    }
+    // If you bump into the wall
+    if(snake[0].x >= 18 || snake[0].x <=0 || snake[0].y >= 18 || snake[0].y <=0){
```
Comment (geekymeeky) [inline Snakegame/snake.js:35]: The snake bumps before it touches the right and bottom wall. 
I think changing this `if(snake[0].x >= 18 || snake[0].x <=0 || snake[0].y >= 18 || snake[0].y <=0)` to `if(snake[0].x > 19 || snake[0].x <=0 || snake[0].y > 19 || snake[0].y <=0)` will resolve the issue.

File: Snakegame/snake.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        return true;
+    }
+        
+    return false;
+}
+
+function gameEngine(){
+    // Part 1: Updating the snake array & Food
+    if(isCollide(snakeArr)){
+        gameOverSound.play();
```
Comment (geekymeeky) [inline Snakegame/snake.js:45]: Music isn't playing.

File: Snakegame/snakegame.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Snake Game</title>
+    <link rel="stylesheet" href="snake.css">
+    <link rel="shortcut icon" href="https://i.pinimg.com/originals/b0/20/57/b020573677505d9d5dde2fdf21e515aa.png" type="image/x-icon">
+</head>
+<body>
+    <div class="body">
```
Comment (geekymeeky) [inline Snakegame/snakegame.html:12]: Add a button to navigate to the homepage.

**Review by geekymeeky — CHANGES_REQUESTED** [review]
Kindly add the requested changes.

File: Snakegame/snake.js (hunk 3 above)
Comment (Subhradeep10) [reply Snakegame/snake.js:45]: Even am unable to fix it, Shall I remove it?

Comment (geekymeeky) [reply Snakegame/snake.js:45]: Yea sure. That would be better.

**Comment by Subhradeep10** [comment]
image

**Comment by geekymeeky** [comment]
@Subhradeep10 I can't play the game. Check if whether it's working or not?

**Comment by Subhradeep10** [comment]
image

Check it now @geekymeeky 

image

**Review by geekymeeky — APPROVED** [review]
Excellent work @Subhradeep10

**Pull request merged**